css/index.css

```diff
@@ -71,6 +72,48 @@ p {
     margin: 0 auto;
 }
 
+header nav{
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    margin: 5% 0;
+    margin-right: 8%;
+}
+
+header nav .nav-btn{
+    text-decoration: none;
+    color: black;
+    font-size: 15px;
+    border: 1px solid white;
+    border-radius: 5px;
+    padding: 2%;
+}
+
```

Comment:
good job utilizing %s in your styles, thats going to come in handy next week with responsive design